recipes/pro-mdnsd/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+import sys
+from conans import ConanFile, CMake, tools
+from conans.tools import download, unzip
+from conans.model.version import Version
+
+
+class mdnsdConan(ConanFile):
+    name = "pro-mdnsd"
+    license = "BSD-3-Clause"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/**"
+    ]
+    homepage = "https://github.com/Pro/mdnsd"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Improved version of Jeremie Miller's MDNS-SD implementation"
+    topics = ("dns", "daemon", "multicast", "embedded", "c")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "cpp_compatible": [True, False]
```

Comment:
This name is not true, c sources are also compatible.
The only difference that compiling as c++ makes, is extra checking + mangling of symbols.

Reply:
I propose a different name, such as `as_cpp` or something similar.

Reply:
would `compile_as_cpp` work better?

Reply:
fine to me.
The original name was misleading.